libs/sdk-ui-kit/src/utils/localeUtilities.ts

```diff
@@ -0,0 +1,8 @@
+// (C) 2020 GoodData Corporation
+
+export function mapChineseLocaleForMoment(intlLocale: string): string {
+    if (intlLocale === "zh-Hans") {
+        return "zh-cn";
+    }
+    return intlLocale;
+}
```

Comment:
this should probably be in the `util` package. as is this fun is also in sdk-ui-filters (search for zh-Hans). both kit and filters already depend on util.

we have translationUtils in the `util` package. the fun could be there.

Reply:
moved to shared location